src/components/itemcontextmenu.js

```diff
@@ -351,14 +351,15 @@ define(["apphost", "globalize", "connectionManager", "itemHelper", "appRouter",
                     document.body.appendChild(textArea);
                     textArea.focus();
                     textArea.select();
-                    try {
-                        document.execCommand("copy");
-
+                    if (document.execCommand("copy")) {
                         require(["toast"], function (toast) {
                             toast(globalize.translate("CopyStreamURLSuccess"));
                         });
-                    } catch (err) {
+                    } else {
```

Comment:
Is there a situation that would actually break the page if this isn't wrapped in a try-catch block? Otherwise, I think this is already good to merge.

Reply:
From the [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand), it doesn't seem to ever return an exception here.
If we merge this, shouldn't we rename the PR as it doesn't fix the original problem? Finally, is the way I added a new string the correct one? Thanks!

Reply:
Renaming the pull request sounds good, and the string addition is indeed correct.

Reply:
Thanks!

Reply:
No problem!